internal/tx_finder/tx_finder.go

```diff
@@ -131,8 +134,9 @@ func (f Finder) GetRawTxs(ctx context.Context, source validator.FindSourceFlag,
 	if source.Has(validator.SourceWoC) && len(ids) > 0 {
 		var wocSpan trace.Span
 		ctx, wocSpan = tracing.StartTracing(ctx, "WocClient_GetRawTxs", f.tracingEnabled, f.tracingAttributes...)
-		wocTxs, err := f.wocClient.GetRawTxs(ctx, ids)
-		defer tracing.EndTracing(wocSpan)
+		var wocTxs []*woc_client.WocRawTx
+		wocTxs, err = f.wocClient.GetRawTxs(ctx, ids)
+		defer tracing.EndTracing(wocSpan, err)
```

Comment:
Should this `EndTracing` be without `defer` here, as this is just the span for `wocClient.GetRawTxs`, similar to TransactionHandler above?